aldjemy/orm.py

```diff
@@ -91,7 +93,7 @@ def _extract_model_attrs(model, sa_models):
 def prepare_models():
     tables = get_tables()
     models = get_django_models()
-    sa_models = getattr(Cache, 'models', {})
+    sa_models = {}
```

Comment:
My mistake. I didn't see it's a part of the public interface. I've rolled this change back. But I had to change the mapping from table_name:sa_model to django_model:sa_model as we can have multiple models on the same table.